src/components/site/EmphasizedImage.md

```diff
@@ -0,0 +1,12 @@
+Emphasized Image is a card with a background image that accepts a description(upper text) and a link(lower text).
+
+```jsx
+<EmphasizedImage
+  backgroundimage={{
+    backgroundImage:
+      "url(//dev.baltimorecountymd.gov/sebin/t/s/homepage-county-executive.jpg)"
+  }}
+  imageheader="County Executive"
+  imagelink="Johnny Olszewski"
```

Comment:
this doesn't seem right, is this image link text? or a subheading?

Reply:
so its a sub and also a link.

Reply:
the word `link` to me suggests "http://ducksarethebest.com". This seems like `subHeader`. We will need both.

Reply:
Okay, I'll add another prop for the href.